### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
+    "connect",
+    "rotate",
+    "file",
+    "minute",
+    "hourly",
+    "daily",
+    "logrotate"
+  ],
+  "license": "MIT",
+  "main": "FileStreamRotator.js",
```
Comment (rahulpatel) [inline package.json]: `main` is now `index.js`

File: index.js [hunk 2]
```diff
@@ -1 +1,196 @@
...
+      if (verbose) {
+        console.log('Rotating file', options.frequency);
+      }
+      var stream = {
+        end: rotateStream.end
+      };
+      stream.write = (function(str, encoding) {
+        var newDate = this.getDate(frequencyMetaData);
+        if (newDate != curDate) {
+          var newLogfile = filename + (curDate ? '.' + newDate : '');
```
Comment (rahulpatel) [inline index.js]: Is this supposed to use `_generateFileName` like above?

**Comment by rahulpatel** [comment]
Hey @Kikobeats,

Sorry it's taken someone so long to take a look at this! I've left a couple comments, nothing to major.

Thanks,
Rahul

**Comment by hxroger** [comment]
@Kikobeats  thanks for the PR. I've pushed to master some changes that fix some issues and implement your changes in a slightly different way using moment.js date formats. Should be pushed to NPM in the next day or two. The change allows to place the date in the filename wherever required and using any format required. Format is not enforced so users need to use a format that matches the rotation required.

**Comment by Kikobeats** [comment]
:)

**Comment by hxpaul** [comment]
Thanks again for the PR. I think @hxroger has addressed your suggestions in a slightly different way so I'll close this. Please feel free to reopen / rebranch from master though.

**Pull request closed without merging**